src/poly/multiopen/prover.rs

```diff
@@ -29,8 +28,8 @@ impl<C: CurveAffine> Proof<C> {
     where
         I: IntoIterator<Item = ProverQuery<'a, C>> + Clone,
     {
-        let x_4: C::Scalar = get_challenge_scalar(Challenge(transcript.squeeze().get_lower_128()));
-        let x_5: C::Scalar = get_challenge_scalar(Challenge(transcript.squeeze().get_lower_128()));
+        let x_4 = ChallengeScalar::<_, ()>::get(transcript);
+        let x_5 = ChallengeScalar::<_, ()>::get(transcript);
```

Comment:
Let's rename these challenges to

`X1`, `X2`, (and `X3`, `X4` below) since the numbering no longer makes sense. Regarding the effective shadowing of challenge names, see my other comment.